package.json

```diff
@@ -34,7 +34,7 @@
     "@types/swagger-schema-official": "~2.0.18",
     "@types/urijs": "~1.19.1",
     "lodash": "^4.17.15",
-    "openapi-schema-to-json-schema": "philsturgeon/openapi-schema-to-json-schema#9692af184c6a3e3d952a88968da07491e47135fb",
+    "openapi-schema-to-json-schema": "stoplightio/openapi-schema-to-json-schema#c8b5f0c74270d505fc39635edef28e09df89601f",
```

Comment:
@philsturgeon we switched to our fork, but if you could create a branch in your repo containing all the commits our branch contains, we would switch back and kill our fork.